qemu/tests/live_backup_base.py

```diff
@@ -213,3 +228,16 @@ def verify_md5s(self):
         """
         file_names = self.params["file_names"]
         return map(self.verify_md5, file_names.split())
+
+    def verify_efficiency(self):
+        if not self.incremental_backup_time < self.full_backup_time:
+            raise exceptions.TestFail("incremental backup time %s, "
+                                      "larger than full backup time %s " %
+                                      (self.incremental_backup_time,
+                                       self.full_backup_time))
+        full_size = self.get_image_size(self.image_chain[0])
+        incre_size = self.get_image_size(self.image_chain[1])
+        if incre_size > full_size:
```

Comment:
Hi Qianqian,
The snapshot image size > base image size is valid.  So the incre_size > full_size should be valid.  Any opinion?

Best Regards,
Pei